pkg/partials/types/rest.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	"github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// PartialRESTPayload is a partial representation of a REST request payload
+// that contains only the fields necessary to generate an error response and
+// handle accounting for the request's method as determined by its headers.
+type PartialRESTPayload struct {
```

Comment:
In the previous PR, left a few comments about the choice of the world "Partial"

Reply:
Replied on the previous PR, I think partial is a good name. Have moved all that logicinto the previosu PR